tests/pytest.ini

```diff
@@ -24,6 +24,7 @@ markers:
     static_config: static_config marker
     dependency: dependency marker
     skip_traffic_test: skip_traffic_test marker
+    stress: stress test marker
```

Comment:
Is the behavior of pytest stress plugin to run a test many times until the time upper limit is reached?
This way of doing stress test is not efficient enough for our test design. Each of the test script has a large overhead of setting up the global fixtures, do sanity check, etc.
Maybe use the confidence level is more appropriate? You can search the code base for examples of confidence level.
Basically the idea to use in-test-script loop to run something really matters many times. This way can avoid the overhead.

Reply:
I will look into it. Unfortunately, there is no way to skip setup and teardown between stress runs. I would think that was the whole idea of the stress test is to do setup, run test N times and then tear down. But pytest-stress doesn't do that. And it appeared to me at that time that I had to manually run it in a loop say 1000 times. I can add an option to pass in the number of loops to run the test; in this case it would translate to number of times the packet is sent/received after the ACL has been setup.

Reply:
Thank you @wangxin. I have reverted the reliance on `pytest-stress` and have relied on using completeness level that other stress tests use.